eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -62,11 +62,11 @@ jobs:
                      -configuration Release \
           | grep PROJECT_FILE_PATH \
           | awk -F ' = ' '{print $NF}' \
-          | while read PROJECT_PATH; do
-              PROJECT_NAME=$(echo $PROJECT_PATH | awk -F/ '{print $NF}' | sed 's/.xcodeproj//')
-              BCSYMBOLMAP_FILE=$(grep -l $PROJECT_NAME.build $(Build.ArtifactStagingDirectory)/AzureSDK-iphoneos.xcarchive/BCSymbolMaps/* | head -n1)
-              echo "Bundling xcframework for $PROJECT_NAME"
-              echo "Including BCSymbolMap at path $BCSYMBOLMAP_FILE"
+            #| while read PROJECT_PATH; do
+            #              PROJECT_NAME=$(echo $PROJECT_PATH | awk -F/ '{print $NF}' | sed 's/.xcodeproj//')
+            #              BCSYMBOLMAP_FILE=$(grep -l $PROJECT_NAME.build $(Build.ArtifactStagingDirectory)/AzureSDK-iphoneos.xcarchive/BCSymbolMaps/* | head -n1)
+            #              echo "Bundling xcframework for $PROJECT_NAME"
+            #              echo "Including BCSymbolMap at path $BCSYMBOLMAP_FILE"
             #   xcodebuild -verbose -create-xcframework \
             #              -framework $(Build.ArtifactStagingDirectory)/AzureSDK-iphonesimulator.xcarchive/Products/Library/Frameworks/$PROJECT_NAME.framework \
             #                -debug-symbols $(Build.ArtifactStagingDirectory)/AzureSDK-iphonesimulator.xcarchive/dSYMs/$PROJECT_NAME.framework.dSYM \
```

Comment:
You should just comment out this entire `-script` step.

Reply:
I was debating if we wanted to remove the whole step...or just removing just to piece causing the issue.
I was thinking since it was was the BCSymbol mapping that we are having trouble with to just disable it. Reading the rest of the I guess it is just a duplicate of `Publish release archive [iphoneos]` so I guess we can just remove the whole thing